package/toltec-bootstrap/toltecctl

```diff
@@ -541,9 +545,9 @@ generate-opkg-conf() {
 # then run \`toltecctl generate-opkg-conf\` to regenerate this file
 
 dest root /
-dest ram /opt/tmp
-lists_dir ext /opt/var/opkg-lists
-option tmp_dir /opt/tmp
+dest ram $toltec_dest/tmp
```

Comment:
same in those 3 lines

Reply:
These lines are being written to a config file, so wrapping them inside `"` would not do what is expected.

Reply:
I'm not sure what you mean, those are arguments to a binary or more likely some function (dest/lists_dir/etc). The "" would not be passed there, they would simply make sure that a param is a single param and whitespace wouldn't split it ($2/$3/etc)

Reply:
Not sure what's happened here, you didn't reply, marked as resolved yet did not change it. So either you assumed I'm wrong or you forgot something.